sema/commons/glob/globbery.py

```
@@ -8,8 +8,8 @@
 
 def getMatchingGlobPaths(
     root: Path,
-    includes: list[str] = ["**/*"],
-    excludes: list[str] = [],
+    includes: list[str] | str = ["**/*"],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)